index.src.html

```diff
@@ -1078,6 +1078,31 @@ <h3 id="unknown-policy-values">Unknown Policy Values</h3>
 
 </section>
 
+<section>
+  <h2 id="iana">IANA Considerations</h2>
+
+  <h3 id="iana-http-headers">HTTP Headers</h3>
```

Comment:
Nit: I think this header is unnecessary, given that there's only one thing we're asking IANA to do. CSP had several, AFAIR, so multiple headers were helpful.